src/TunTap.ts

```diff
@@ -66,7 +66,7 @@ export class TunTap {
         };
         
         process.once('exit', cleanup);
-        process.once('SIGINT', cleanup);
+        process.once('SIGINT', () => { cleanup(); process.exit(0); });
```

Comment:
do we still need to call process exit explicitly?

Reply:
ah, it is not required anymore,
Thank you for review mykola. updated with commit https://github.com/appium/appium-ios-tuntap/pull/1/commits/87785c86ac425c92cfe9b71ff5734446e79e05c1